plugin/evm/vm_test.go

```diff
@@ -3204,3 +3219,9 @@ func TestSignatureRequestsToVM(t *testing.T) {
 	err = vm.Network.AppRequest(context.Background(), ids.GenerateTestNodeID(), 1, time.Now().Add(60*time.Second), requestBytes)
 	require.NoError(t, err)
 }
+
+func createValidatorPrivateKey(keyFile string) {
+	// Create a new validator private key file
+	privateKey := []byte("31b571bf6894a248831ff937bb49f7754509fe93bbd2517c9c73c4144c0e97dc")
+	os.WriteFile(defaultValidatorPrivateKeyFile, privateKey, 0644)
```

Comment:
maybe we could write it to `/tmp/validator_private_key.pk` then we don't have to handle deletetion everytime

Reply:
done